app/views/govuk_component/docs/search.yml

```diff
@@ -0,0 +1,19 @@
+name: "Search"
+description: "Search box"
```

Comment:
Other things that should be indicated upfront:
* Component must be included within a form
* Search input uses the name `q`, which isn't customisable (that's ok for now I think)